src/app/core/core.module.ts

```diff
@@ -30,7 +31,13 @@ export const COMPONENTS = [
     ],
     declarations: COMPONENTS,
     exports: COMPONENTS,
-    providers: [TransactionBackendService, DurationAsDaysPipe, BaseResolver]
+    providers: [BaseResolver]
 })
 export class CoreModule {
+    static forRoot(): ModuleWithProviders {
```

Comment:
This will make the services in line nr 40 to be singleton, despite the fact that CoreModule is imported in multiple modules - The common singleton instace will be shared